phoenix-core/src/main/java/org/apache/phoenix/index/PhoenixTransactionalIndexer.java

```diff
@@ -0,0 +1,497 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements. See the NOTICE
```

Comment:
Check that license matches others - looks like it might have gotten formatted incorrectly.